### PR TITLE
Fourth milestone

### PR DESCRIPTION
**Pull request by orestis-kaplan**

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  if (operation === '+') {
+    console.log(numberOne, numberTwo);
```
Comment (SunnyAsar) [inline src/logic/operate.js]: Please remove the log message

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    return Big(numberOne).plus(numberTwo).toString();
+  }
+  if (operation === '-') { return Big(numberOne).minus(numberTwo).toString(); }
+  if (operation === '÷') { return Big(numberOne).div(numberTwo).toString(); }
+  if (operation === 'x') { return Big(numberOne).times(numberTwo).toString(); }
```
Comment (SunnyAsar) [inline src/logic/operate.js]: this is a good place to use switch instead of multiple `if`

File: src/logic/calculate.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  let { total } = data;
+  let firstNumber = 0;
+  let secondNumber = 0;
+  let negative = false;
+  let symbol = '';
+
+
+     if (buttonName === 'AC') {
+       total = ''; 
+    } else if (buttonName === '+/-') {
```
Comment (SunnyAsar) [inline src/logic/calculate.js]: The Linting here is not right. its a little too indented.

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hi @userman95 
good job with this milestone, lets correct a few thing.
- remove commnets and log message,
- There are linting problems in the code base, i have pointed out a couple.
- please use array de structuring whenever possible.
- #### Consider grouping your css files in another directory, they are not components after all

#### Please request another review with the changes

**Review by BalintMendli — APPROVED** [review]
Hi @userman95 ,

All good for this milestone.

### 👍

**Pull request merged**